### PR TITLE
Adding OWNERS file for drupal chart

### PR DESCRIPTION
**Pull request by mattfarina**
@prydonius does this pattern work for a bitnami owners file setup or should we alter something else as well?

### CODE REVIEW

File: stable/drupal/OWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+approvers:
```
Comment (prydonius) [inline stable/drupal/OWNERS:1]: do we need to specify both approves and reviewers? surely an approver supersedes a reviewer?

File: stable/drupal/OWNERS [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+- bitnami-bot
```
Comment (prydonius) [inline stable/drupal/OWNERS]: I would say bitnami-bot should just be a reviewer, or not even in this list

Comment (mattfarina) [reply stable/drupal/OWNERS]: I can remove from the list

File: stable/drupal/OWNERS (hunk 1 above)
Comment (mattfarina) [reply stable/drupal/OWNERS:1]: There's a plugin (that I'll get enabled for us) that requests PR reviews by people listed as reviewers. So, we should keep both sections. Does not request reviews from approvers (so it won't bubble up to the top).

Comment (prydonius) [reply stable/drupal/OWNERS:1]: ah, good to know, thanks!

**Pull request merged**